code/modules/mob/living/simple_animal/friendly/mouse.dm

```diff
@@ -38,7 +38,7 @@
 	icon_state = "mouse_[body_color]"
 	icon_living = "mouse_[body_color]"
 	icon_dead = "mouse_[body_color]_dead"
-	if(prob(75))
+	if(prob(100))
```

Comment:
ARGH, just remove prob check and bump everything down from the indent instead of running a prob check that will always pass